### DISCUSSION TITLE
What is React-Navigations commitment to supporting Web moving forwards?

### DISCUSSION THREAD

**Discussion by jonueki** [react-navigation/react-navigation#11330] [General]
Is the React Navigation team planning on supporting Web for the foreseeable future? The docs says its still [experimental](https://reactnavigation.org/docs/web-support/) so Im wondering if there is a risk for the team to abandon support in v7+

**Context**: My team maintains a React web app and a React native app. We're looking to reduce the complexity of the codebase by re-using many of the same components with the help of React-Native-Web

Right now our Web app is using an old version of React Router (v4) and we are debating on whether to upgrade React Router to (v6) or use React Navigation instead on web. Our mobile app already uses React Navigation.

**Comment by billouboq** [comment]
React navigation works well on web, we use it at Beedeez from version 4 to the latest 6 in web and in native app
